Packs/Slack/Integrations/Slack/Slack.py

```diff
@@ -1945,14 +1795,16 @@ def init_globals(command_name: str = ''):
             demisto.info(f'setting _default_executor on loop: {loop} id: {id(loop)}')
             loop.set_default_executor(concurrent.futures.ThreadPoolExecutor(max_workers=4))
 
-    BOT_TOKEN = demisto.params().get('bot_token', '')
-    ACCESS_TOKEN = demisto.params().get('access_token', '')
+    BOT_TOKEN = demisto.params().get('bot_token', {}).get('password', '')
+    ACCESS_TOKEN = demisto.params().get('access_token', {}).get('password', '')
+    APP_TOKEN = demisto.params().get('app_token', {}).get('password', '')
```

Comment:
So we need 3 tokens? Currently our flow provides 2.

Reply:
We do. With SlackV2 everyone was using the Demisto slack app (which had its own token running on oproxy). Since we are migrating from that, customers will need to provide their own slack app token. It's part of the shift towards "Bring-Your-Own-App" we are implementing.